### ISSUE TITLE
Theme disables spell check highlighting 

### ISSUE THREAD

**Issue by JohnAdamsII** [tomasiser/vim-code-dark#33]
when I have the theme enabled in my vimrc and I run the command :set spell, I am unable to see spell check highlighting when I disable the theme, I see the spell check highlight as expected, OS: ubuntu 18.04 LTS

**Comment by dunstontc** [comment]
You're gonna want to change [these](https://github.com/tomasiser/vim-code-dark/blob/811fcff1526f330e87f3d7663fe60588b22e5dd8/colors/codedark.vim#L161) if you want different styling for spelling errors:
```
call <sid>hi('SpellBad', s:cdNone, s:cdNone, 'undercurl', {})
call <sid>hi('SpellCap', s:cdNone, s:cdNone, 'undercurl', {})
call <sid>hi('SpellLocal', s:cdNone, s:cdNone, 'undercurl', {})
```
As it is right now, spelling errors are just underlined.

**Comment by tomasiser** [comment]
Thank you all! Solved by #52.
